test/hotspot/jtreg/gtest/CompressedKlassGtest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I don't know why you need this.  I thought the gtests just ran with TEST_VM.
Also Copyright SAP is probably wrong for you now.

Reply:
> I don't know why you need this. I thought the gtests just ran with TEST_VM.

I want to see the CompressedKlass tests succeeding with these options: -Xshare:off etc. Because that triggers zero based encoding, and that made the error appearant. Otherwise, the error was hidden, because we usually run with encoding base == start-of-CDS (aka start of Klass range).

> Also Copyright SAP is probably wrong for you now.

Good catch. Its completely wrong now.